source/python/neuropod/backends/caffe/executor.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020 UATC, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import numpy as np
+import os
+import sys
+
+import caffe
+
+from neuropod.backends.neuropod_executor import NeuropodExecutor
+
+
+class CaffeNeuropodExecutor(NeuropodExecutor):
+    """
+    Executes a Caffe neuropod
+    """
+    def __init__(self, neuropod_path):
+        """
+        Load a Caffe neuropod
+
+        :param  neuropod_path:      The path to a Caffe neuropod package
+
+        """
+        super(CaffeNeuropodExecutor, self).__init__(neuropod_path)
+        # Create a folder to store the model
+        neuropod_data_path = os.path.join(neuropod_path, "0", "data")
```

Comment:
Why do we need 0 here?

Reply:
Follow the previous (other framework ) code style